blockchain/api.go

```diff
@@ -91,6 +91,14 @@ type TokenAPI interface {
 	AllowanceOf(ctx context.Context, from, to common.Address) (*big.Int, error)
 	// TotalSupply - all amount of emitted token
 	TotalSupply(ctx context.Context) (*big.Int, error)
+	// IncreaseApproval increase the amount of tokens that an owner allowed to a spender
+	// approve should be called when current allowance = 0. To increment
```

Comment:
What happens if I'll call this func with non-zero current approval?

Reply:
it should increased allowance to spender by sended value